dbt/include/spark/macros/materializations/incremental/incremental.sql

```diff
@@ -2,7 +2,7 @@
 
   {#-- Validate early so we don't run SQL if the file_format + strategy combo is invalid --#}
   {%- set raw_file_format = config.get('file_format', default='parquet') -%}
-  {%- set raw_strategy = config.get('incremental_strategy', default='append') -%}
+  {%- set raw_strategy = config.get('incremental_strategy') or 'append' -%}
```

Comment:
Do we actually need the `or 'append'`? Or can we fall back to `append` being the "true" default, defined in `dbt-core`, when `incremental_strategy` is not defined?

Reply:
At this point we do need the 'append' fallback, until we convert the rest of the code. If we had made the 'default' of the incremental_strategy config attribute 'default' we wouldn't need it, but I still don't like actual string defaults that will show up in every node...

Reply:
Ah, right! Until we convert to add the new method calls here in the incremental materialization. I follow you

Reply:
sounds good to me